apps/spruce/package.json

```diff
@@ -26,7 +26,7 @@
     "dev": "env-cmd -e devLocal yarn start",
     "eslint:fix": "yarn eslint:strict --fix",
     "eslint:staged": "STRICT=1 eslint --concurrency=auto",
-    "eslint:strict": "STRICT=1 eslint --max-warnings=62 --concurrency=auto .",
+    "eslint:strict": "STRICT=1 eslint --max-warnings=61 --concurrency=auto .",
```

Comment:
This seems a little brittle

Reply:
I think Sophie's goal is to continue decreasing this down until we reach 0, at that point would mean we won't be able to introduce new warnings into the codebase